packages/compose/src/higher-order/pure/test/index.js

```diff
@@ -74,6 +84,9 @@ describe( 'pure', () => {
 		);
 
 		const { rerender } = render( <MyComp /> );
+		expect( console ).toHaveWarnedWith(
+			'wp.compose.pure is deprecated since version 6.5. Please use Use `memo` or `PureComponent` instead instead.'
```

Comment:
What is PureComponent?

Reply:
https://react.dev/reference/react/PureComponent